hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -201,10 +227,27 @@ public:
    /// Get the sum of weights.
    Weight_t GetSumOfWeights() const { return fSumWeights; }
 
-   /// Merge with other RHistStatTotalSumOfWeights data, assuming same bin configuration.
+   /// Add an other RHistStatTotalSumOfWeights data, assuming same bin configuration.
    void Add(const RHistStatTotalSumOfWeights& other) {
       fSumWeights += other.fSumWeights;
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      fSumWeights *= scalar;
+   }
+
+   /// Divide by an other RHistStatTotalSumOfWeights data, assuming same bin configuration.
+   void Divide(const RHistStatTotalSumOfWeights& other) {
+      fSumWeights /= other.fSumWeights;
+   }
+
+   /// Divide by an other RHistStatTotalSumOfWeights data, assuming same bin configuration.
+   void DivideBinomial(const RHistStatTotalSumOfWeights& other) {
+      // /!\ Should be the sum of the updated weights (after division) of the bin contents
```

Comment:
Out of curiosity: what's `/!\` Looks great for a warning sign, does Doxygen understand that? And maybe:
```suggestion
      // /!\ Must be called *after* the bin contents have been divided!
```
(if that's what it is). More occurrences below.